qtcodes/fitters/lattice_decoder.py

```diff
@@ -425,6 +425,7 @@ def draw(
         dpi: Optional[int] = None,
         node_size: Optional[int] = None,
         font_size: Optional[float] = None,
+        show: Optional[bool] = True,
```

Comment:
Please add to the doc string to reflect this change.

Reply:
Fixed